modules/clang-cxx-dev-tools.cmake

```diff
@@ -0,0 +1,43 @@
+# http://www.labri.fr/perso/fleury/posts/programming/using-clang-tidy-and-clang-format.html
+
+# Additional target to perform clang-format/clang-tidy run
+# Requires clang-format and clang-tidy
+file(GLOB_RECURSE
+        ALL_CXX_SOURCE_FILES
+        ${CMAKE_CURRENT_SOURCE_DIR}/Telegram/SourceFiles/*.cpp
```

Comment:
I'd prefer this list passed in from the outside, because this is definitely not the full list.